### PR TITLE
Upgrade service bus SDK

### PR DESCRIPTION
**Pull request by ErikMogensen**
Switch from using Microsoft.Azure.ServiceBus to Azure.Messaging.ServiceBus.

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpgradeServiceBusSdk.8.build.751 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37075282) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/27da5eb688 by @ErikMogensen)

File: src/Common/Helpers/ServiceBusHelper.cs [hunk 1]
```diff
@@ -34,14 +34,17 @@
...
 using Microsoft.ServiceBus.Messaging;
+
 using ServiceBusExplorer.Enums;
 using ServiceBusExplorer.Helpers;
 using ServiceBusExplorer.ServiceBus.Helpers;
 using ServiceBusExplorer.Utilities.Helpers;
+
 using AzureNotificationHubs = Microsoft.Azure.NotificationHubs;
-using NewSdkManagement = Microsoft.Azure.ServiceBus.Management;
+using NewSdkManagement = Azure.Messaging.ServiceBus.Administration;
```
Comment (SeanFeldman) [inline src/Common/Helpers/ServiceBusHelper.cs:47]: As we're not mixing the two SDKs together, is this aliasing necessary?

File: src/Common/Helpers/ServiceBusHelper.cs [hunk 2]
```diff
@@ -5294,40 +5297,32 @@ public ServiceBusHelper2 GetServiceBusHelper2()
...
             serviceBusHelper2.TransportType = UseAmqpWebSockets
-                ? Microsoft.Azure.ServiceBus.TransportType.AmqpWebSockets
-                : Microsoft.Azure.ServiceBus.TransportType.Amqp;
+                ? Azure.Messaging.ServiceBus.ServiceBusTransportType.AmqpWebSockets
+                : Azure.Messaging.ServiceBus.ServiceBusTransportType.AmqpTcp;
             return serviceBusHelper2;
         }
 
-        public async Task<NewSdkManagement.QueueDescription> GetNewSdkQueueDescription(QueueDescription oldQueueDescription)
+        public async Task<NewSdkManagement.QueueProperties> GetNewSdkQueueProperties(QueueDescription oldQueueDescription)
```
Comment (SeanFeldman) [inline src/Common/Helpers/ServiceBusHelper.cs:5305]: GetNewSdkQueueProperties --> GetQueueProperties

File: src/Common/Helpers/ServiceBusHelper.cs [hunk 3]
```diff
@@ -5294,40 +5297,32 @@ public ServiceBusHelper2 GetServiceBusHelper2()
...
-            var managementClient = new NewSdkManagement.ManagementClient(connectionString);
-            try
-            {
-                return await managementClient.GetQueueAsync(oldQueueDescription.Path);
-            }
-            finally
-            {
-                await managementClient.CloseAsync();
-            }
+            var managementClient = new NewSdkManagement.ServiceBusAdministrationClient(connectionString);
```
Comment (SeanFeldman) [inline src/Common/Helpers/ServiceBusHelper.cs:5307]: Should we be consistent and adopt the new SDK naming?
If yes, `managementClient` --> `administrationClient`.

**Review by SeanFeldman — CHANGES_REQUESTED** [review]

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpgradeServiceBusSdk.9.build.753 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37094873) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/54136ce1f4 by @ErikMogensen)

File: src/Common/Helpers/ServiceBusHelper.cs (hunk 2 above)
Comment (ErikMogensen) [reply src/Common/Helpers/ServiceBusHelper.cs]: Changed

File: src/Common/Helpers/ServiceBusHelper.cs (hunk 3 above)
Comment (ErikMogensen) [reply src/Common/Helpers/ServiceBusHelper.cs]: Yes, that is better. Changed it.

**Comment by paolosalvatori** [comment]
@ErikMogensen did you remove the Microsoft.Azure.ServiceBus everywhere, or just in some files? What about the HandleTopicControl for example?

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpgradeServiceBusSdk.12.build.756 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37096177) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/0c655bbaa6 by @ErikMogensen)

**Comment by ErikMogensen** [comment]
Yes, @paolosalvatori I removed Microsoft.Azure.ServiceBus everywhere. I dreaded a situation where we would have three Service bus SDKs. Removing the oldest SDK is quite a big task though since it is heavily used and there are message format incompatibilities including the WCF support.

**Comment by paolosalvatori** [comment]
Does this mean that WCF format is no more supported? Just asking. At this point, we can also implement Azure AD authentication.

**Review by paolosalvatori — APPROVED** [review]

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 5.1.0-UpgradeServiceBusSdk.14.build.760 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/37100612) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/9634ee8bbf by @ErikMogensen)

**Comment by ErikMogensen** [comment]
@paolosalvatori,  as I understand it WCF is not supported by the latest SDK. However, it is still supported in the current version of SBE since almost all Service Bus code in it is still using the oldest SDK.

**Comment by ErikMogensen** [comment]
Also @paolosalvatori , I think you'll have to approve this one again since I made some minor changes.

**Review by SeanFeldman — APPROVED** [review]

**Comment by paolosalvatori** [comment]
No worries, as I said in another comment I don't think there's no one out there that still uses WCF. We can remove WCF support from SBE eventually.

**Pull request merged**